tests/launch.rs

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: Apache-2.0
+
+#[cfg(feature = "openssl")]
```

Comment:
Why not make the whole file:
```rust
#![cfg(feature = "openssl")]
```

Reply:
![](https://media.giphy.com/media/yqtpq8rqqXBh6/giphy.gif)